scripts/templates/diag_table_template

```diff
@@ -31,6 +33,7 @@
  "ocean_model", "ePBL_h_ML", "ePBL",     "ocn%4yr%2mo%2dy%2hr","all",.true.,"none",2
  "ocean_model", "MLD_003",   "MLD_003",  "ocn%4yr%2mo%2dy%2hr","all",.true.,"none",2 
  "ocean_model", "MLD_0125",  "MLD_0125", "ocn%4yr%2mo%2dy%2hr","all",.true.,"none",2
+ "ocean_model", "vmo",       "vmo",      "ocn%4yr%2mo%2dy%2hr","all",.true.,"none",2
```

Comment:
I looked up the meaning of vmo: 
"ocean_model", "vmo"  
    ! long_name: Ocean Mass Y Transport
    ! units: kg s-1
    ! standard_name: ocean_mass_y_transport
    ! cell_methods: xh:sum yq:point zl:sum

Should we also be adding "umo"? for the x transport?

Reply:
@jkbk2004 This is the only unresolved issue. 

Let me know when you are done with this, and I will test two cases through the workflow (3dvar and fcst_only), and merge the code.